source/math/IntRandomGenerator.ooc

```diff
@@ -15,17 +15,35 @@
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
 
 use ooc-math
+import math
 import os/Time
 
-IntRandomGenerator: class {
-	_backend: FloatRandomGenerator
-	init: func ~withBackend (=_backend)
-	free: override func {
-		this _backend free()
-		super()
+IntUniformRandomGenerator: class {
+	_state, _min, _max, _range: Int
+	init: func (seed := Time microtime()) {
+		this _min = 0
+		this _max = 32767
```

Comment:
This is 0x7FFF (line 35) giving the maximum range that next~fast() can return, but I guess we could go with something else?